docs/docusaurus.config.ts

```diff
@@ -43,6 +44,18 @@ const config: Config = {
       {
         excludeAliases: ['console']
       }
+    ],
+    [
+      'client-redirects',
+      {
+        createRedirects(routePath) {
+          if (routePath.includes('/entra')) {
```

Comment:
Shouldn't we be redirecting from aad to entra rather than the other way around, or am I reading this incorrectly?

Reply:
What @Jwaegebaert does here is register the aad redirect for each path that has entra in it. It looks odd, but it's correct...

Reply:
It's counter intuitive indeed. But during build, this piece of code will generate a redirect page for each page that contains `entra`.